### ISSUE TITLE
Pin a suggestions request thread

### ISSUE THREAD

**Issue by qdacsvx** [iftechfoundation/ifdb-suggestion-tracker#80]
Make an issue for people to request suggestions and pin it.

**Comment by dfabulich** [comment]
I'm not sure I follow what you have in mind here.

Would you want this to be pinned on the IFDB site? That sounds something like #24.

Or do you want us to pin an issue here on this tracker? (But, if you've found your way here, you'll probably figure out that you can file a suggestion…?)

Or would you like us to in a thread on intfiction.org? (At the top of the whole site? Or just a category? … which category?)

**Comment by qdacsvx** [comment]
I was thinking of pinning an issue here. It's easier to write 10 ideas than to make suggestions. So this is a suggestion (idea) solicitation method.

**Comment by curiousdannii** [comment]
I disagree with this. Each suggestion should be posted separately so that they can be clearly and uniquely referred to.

**Comment by curiousdannii** [comment]
If we want more freeform discussions, Github does have a discussions feature now: https://docs.github.com/en/discussions/quickstart

But I don't know if we'd want to split discussions between here and intfiction.org. I could make a public subcategory for IFDB discussions if that would help.

**Comment by qdacsvx** [comment]
>I disagree with this. Each suggestion should be posted separately so that they can be clearly and uniquely referred to.

My point is that some people can't do what you suggest @curiousdannii. There are many readers but few writers. I agree with your comment as an ideal. Github is a macho culture. We can't force shy people to participate. Some helper is needed to turn the ideas of shy people into suggestions #109.

Maybe a Questions thread is better.
